### PR TITLE
Update create-secure-guest-sharing-environment.md

### PR DESCRIPTION
**Pull request by msbemba**
Updated the section Set up web-only access for guests to reduce ambiguity

per issue#https://github.com/MicrosoftDocs/microsoft-365-docs/issues/10161

### CODE REVIEW

**Comment by msbemba** [comment]
@JohanFreelancer9 Please copy review

File: microsoft-365/solutions/create-secure-guest-sharing-environment.md [hunk 1]
```diff
@@ -149,7 +149,7 @@ It's important to note that for SharePoint and OneDrive locations, documents wil
 
 ## Set up web-only access for guests
 
-You can require guests to access your teams, sites, and files by using a web browser only. This reduces the chance that they might download sensitive files and leave them on an unmanaged device. This is also useful when sharing with environments that use shared devices.
+You can require guests to not access your teams, sites, and files from non-compliant devices using non-browser client applications. Instead you'd want them to use web browser only. This reduces the chance that they might download sensitive files and leave them on an unmanaged device. This is also useful when sharing with environments that use shared devices.
```
Comment (JohanFreelancer9) [inline microsoft-365/solutions/create-secure-guest-sharing-environment.md]:
```suggestion
You might require guests to not access your teams, sites, and files from non-compliant devices using non-browser client applications. Instead, you would want them to use a web browser only. This reduces the chance that they might download sensitive files and leave them on an unmanaged device. It is also useful when sharing with environments that use shared devices.
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review.

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by yogkumgit** [comment]
@MikePlumleyMSFT Please merge this PR. Thanks

**Pull request merged**